### PR TITLE
2024 advent calendar, Weston Thayer "aria-labelledby = self"

### PR DESCRIPTION
**Pull request by WestonThayer**

### CODE REVIEW

File: hell/adventcalendar/2024/6/index.md [hunk 1]
```diff
@@ -2,51 +2,69 @@
...
-    url: "#"
-    link_label: "your-awesome-website.com/blog"
-  - label: "yx on ABC"
-    url: "https://abc-social-media-site"
-    link_label: "@xy"
+  - label: "Website"
+    url: "https://assistivlabs.com"
+    link_label: "assistivlabs.com"
 active: true
 intro: "<p>Short description of the post</p>"
```
Comment (WestonThayer) [inline hell/adventcalendar/2024/6/index.md:12]: Let me know if this is used anywhere, I can add it.

File: hell/adventcalendar/2024/6/index.md [hunk 2]
```diff
@@ -2,51 +2,69 @@
...
+## Judgement
+
+As is, I'd feel like I was nitpicking by requesting the unnecessary `aria-labelledby` to be removed.
+
+But imagine this is in a large codebase with many developers working on it. Consider how easy it would be for someone tasked with adding a notification badge for example to the link to miss the implications of `aria-labelledby`:
+
+```html
+<a href="/updates" aria-labelledby="contents">
+  <span id="contents">Latest updates</span>
+  <span class="notification-badge">{{ updates.length }}</span>
```
Comment (WestonThayer) [inline hell/adventcalendar/2024/6/index.md:54]: Is there a way to escape the `{{` so it renders?

Comment (WestonThayer) [reply hell/adventcalendar/2024/6/index.md:54]: @matuzo if you get a chance. The curly braces don’t render to HTML as is

File: hell/adventcalendar/2024/6/index.md (hunk 1 above)
Comment (matuzo) [reply hell/adventcalendar/2024/6/index.md:12]: At the bottom of the page, like here: https://htmhell.dev/adventcalendar/2023/2/

File: hell/adventcalendar/2024/6/index.md (hunk 2 above)
Comment (matuzo) [reply hell/adventcalendar/2024/6/index.md:54]: Fixed in the advent24 branch

**Pull request merged**